ApolloExtra.lua

```diff
@@ -2602,9 +2852,9 @@ OlympusLootData.ApolloUpgrade = {
 					AllAtLeastRarity = "Rare",
 					HasAtLeastRarity = "Epic",
 				},
-				{ Cue = "/VO/Apollo_0032",
+				{ Cue = "/VO/Apollo_0081",
 					StartSound = "/Leftovers/World Sounds/MapZoomInShort", UseEventEndSound = true,
-					Text = "I have been waiting for a special moment to confess my great appreciation for your deeds... this moment's special enough, isn't it?" },
+					Text = "I gathered all my god-power to deliver this. Don't let it go to waste, Zagzag. {#DialogueItalicFormat}Yawn{#PreviousFormat}, I'll go rest now. You can do this." },
```

Comment:
"I gathered all my god-power to deliver this. Don't let it go to waste, Zagzag. {#DialogueItalicFormat}Yawn{#PreviousFormat}, I think I'll take a nap. You can do this!"